.github/CODEOWNERS

```diff
@@ -16,8 +16,8 @@
 /packages/nimble-components @rajsite @jattasNI
 /packages/nimble-components/.storybook @rajsite @jattasNI @fredvisser
 /packages/nimble-components/src/wafer-map @rajsite @jattasNI @DStavilaNI @zszilagy
-/packages/nimble-components/src/rich-text-viewer @rajsite @jattasNI @vikisekarNI
-/packages/nimble-components/src/rich-text-editor @rajsite @jattasNI @vikisekarNI
+/packages/nimble-components/src/rich-text-viewer @rajsite @jattasNI @vivinkrishna-ni
```

Comment:
Added @vivinkrishna-ni to PR to approve being CODEOWNER for rich-text components (or let us know who should be owner).

Reply:
Sure, I can be a CODEOWNER for the changes related to rich test components.

I think the folder path needs to be updated to `src/rich-text` and `src/rich-text-mention` in this file.

Reply:
Sounds good! Thanks!